src/validation-errors/unevaluated-prop.js

```diff
@@ -0,0 +1,32 @@
+import BaseValidationError from './base';
+
+export default class UnevaluatedPropValidationError extends BaseValidationError {
```

Comment:
`UnevaluatedPropValidationError` is a bit of a mouthful; a comment explaining what it signifies somewhere would be nice 😬

Reply:
lol no docs here https://json-schema.org/understanding-json-schema/reference/object.html#unevaluated-properties

Reply:
Would `UnevaluatedPropertyError` work? or does it need to include validation for some reason?

Reply:
Unfortunately this is the same format for these error handling classes that's already present:

* RequiredValidationError
* AdditionalPropValidationError
* EnumValidationError
* DefaultValidationError
* UnevaluatedPropValidationError